README.md

```diff
@@ -99,3 +100,6 @@ npm test
 ## License
 
 This project is distributed under the MIT license.
+
+[appveyor-badge]: https://ci.appveyor.com/api/projects/status/7owf61lo8uujbjok/branch/master?svg=true
```

Comment:
FYI, just checked you are an owner, in theory you should also have owner privs on appveyor. Although the permission syncing appears funky at times.

Enabling it now!